slides/2-html-css.html

```diff
@@ -566,12 +568,87 @@ <h2><i class="fas fa-paint-brush"></i> Introduction to CSS</h2>
                         
                         - CSS3 is the latest version of the standard.
-                        
+
                         https://developer.mozilla.org/en-US/docs/Web/CSS
                         
                     </textarea>
                 </section>
-                <section data-markdown style="font-size: 0.6em;">
+                <section data-markdown style="font-size: 0.55em;">
+                    <textarea data-template>
+                        ## <i class="fas fa-paint-brush"></i> Anatomy of a CSS document
+
+                        A CSS document is a sequence of **rulesets**, each
+                        applying declarations (i.e. styling descriptions) to
+                        *specific* parts of the document.
+
+                        <img src="images/css_rulesets.png" />
+
+                        A ruleset is composed of a comma-separated list of
+                        **selectors** and a **declarations block** describing
+                        the styling of the selected elements.
+
+                    </textarea>
+                </section>
+                <section data-markdown style="font-size: 0.55em;">
+                    <textarea data-template>
+                        ## <i class="fas fa-paint-brush"></i> Anatomy of a CSS document
+
+                        A declaration block is a sequence of **declarations**,
+                        each consisting of a **property** and the **value**
+                        associated to that property, separated by a colon
+                        (`:`). There are more than [100 different
+                        properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference)
+                        in CSS.
+
+                        <img src="images/css_declarations_block.png" />
+
+                        The **cascading** behavior of CSS means that a
+                        declaration overrides any conflicting declarations made
+                        earlier.
+
+                        https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax
+
+                        Notes:
+
+                        - **Selectors** are conditions selecting some elements of the page, those that will be affected by the ruleset.
+                        - **Declarations** are grouped in **blocks**, and separated by semi-colons (`;`). They can sometimes be nested in the case of media queries (will be covered later in this presentation).
+                        - **Properties** and **values** form **declarations** and are separated by a colon (`:`). They are case-insensitive. There are more than [100 different properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference) in CSS.
+
+                    </textarea>
+                </section>
+                <section data-visibility="hidden" data-markdown style="font-size: 0.6em;">
```

Comment:
This slide and the next are hidden, their content is reorganized into the slides above.